src/code/jpegdecoder.c

```diff
@@ -2,6 +2,10 @@
 #include "stdbool.h"
 #include "macros.h"
 
+s32 JpegDecoder_ProcessMcu(JpegHuffmanTable* hTable0, JpegHuffmanTable* hTable1, u16* mcu, s16* unk);
+s32 JpegDecoder_ParseNextSymbol(JpegHuffmanTable* hTable, s16* outCoeff, s8* outZeroCount);
+u16 JpegDecoder_ReadBits(u8 len);
```

Comment:
I'm not a huge fan of declaring prototypes in C files instead of on a header since you lose the compiler checking the declaration actually matches the function definition (like the OoT, where the crash debugger is called with wrong parameters)

Reply:
I agree, these are forward declarations for functions in this same file though :)